src/SwedbankPay.Sdk/PaymentInstruments/Invoice/ApprovedLegalAddress.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    public class ApprovedLegalAddress
```

Comment:
From its use, I think `Addressee` is a good name for this class?

```suggestion
    public class Addressee
```